torch/profiler/profiler.py

```diff
@@ -45,6 +45,33 @@ def _default_schedule_fn(_: int) -> ProfilerAction:
     """
     return ProfilerAction.RECORD
 
+def tensorboard_trace_handler(dir_name: str, worker_name: Optional[str] = None):
+    """
+    Outputs tracing files to directory of ``dir_name``, then that directory can be
+    directly delivered to tensorboard as logdir.
+    ``worker_name`` is used to distinguish different workers in distributed scenario,
+    it will be set to '[hostname]_[pid]' by default.
+
+    .. note::
+        Example of using Tensorboard plugin for pytorch:
+
+        - pip install -i https://test.pypi.org/simple/ tensorboard-plugin-torch-profiler==0.1.0
```

Comment:
remove this since it is still on test pypi.
we may update the doc after the package is published to pypi.